#1 migrate infura endpoint to package (#1)
slingr-stack/infura-package: README.md

> <table>

Reviewer: ```suggestion
<table class="table" style="margin-top: 10px">
```

    </thead>
    <tbody>
    <tr>
        <td>Infura package</td>
        <td>December 28, 2023</td>
        <td>Detailed description of the API of the Infura package.</td>
    </tr>
    </tbody>
</table>

# Overview

> The Infura endpoint has the following features:

Reviewer: ```suggestion
The Infura package has the following features:
```

- Authorization and generation of ApiKey tokens
- Access to the whole REST API

Please make sure you take a look at the documentation from Infura as features are based on its API:

- [API Reference](https://docs.infura.io/networks/ethereum/json-rpc-methods)

# Javascript API

The Javascript API of the infura package has three pieces:

- **HTTP requests**
- **Flow steps**

## HTTP requests

> You can make `GET`,`PUT`,`PATCH`,`DELETE` requests to the [infura API](API_URL_HERE) like this:

Reviewer: ```suggestion
You can make `GET`,`PUT`,`PATCH`,`DELETE` requests to the [infura API](https://docs.infura.io/networks/ethereum/json-rpc-methods) like this:
```

```javascript

> var response = pkg.infura.api.get('/path3')

Reviewer: Review this calls, if these does not adapt to the actual call, remove and use the same example of ths PR.

## Dependencies

> * HTTP Service (Latest Version)

Reviewer: ```suggestion
* HTTP Service (Latest v1.3.7)
```
